### DISCUSSION TITLE
Top vs. Bottom vs. Same as Backend: Where should we display newly added tasks?

### DISCUSSION THREAD

**Discussion by marbetschar** [elementary/tasks#220] [Ideas]
As we are supporting different backends, we'll have different properties to sort with. For example, in GTasks we to use the `X-EVOLUTION-GTASKS-POSITION` property, for WebDAV backends its `X-APPLE-SORT-ORDER`.

If one uses multiple backends this can lead to different sort behavior of newly added tasks:

![113402492-5f23ca00-93a5-11eb-9c95-1f8c1c40523b](https://user-images.githubusercontent.com/392542/114588278-ca637b00-9c86-11eb-9de5-85d1ad39d3d2.png)

This raises the question, how we want to sort in elementary Tasks in terms of UX?

Do we want the sorting to adhere to whatever is used in the corresponding backend and be therefore potentially different within the app - or to we want to use the sort behaviour to be consistent in the elementary Tasks app, but it then may differ from what is used in the corresponding backend? And if we use the same behaviour regardless of the backend: Should new tasks be displayed at the top of the list - or at the end?

**Comment by leodom** [comment]
For my user case, manual sorting is the feature I need :)
Next would be to sort tasks by last one added showing first. 
Where is the data stored ? Any txt/Json/XML file that can be tampered to sort 'manually' ?
